snapwell/wellpath.py

```diff
@@ -14,11 +14,37 @@
 
 import logging
 from functools import wraps
+from math import inf, isinf, isnan
 from os.path import exists
 
-from .snap_utils import Inf, close, finiteFloat
-from .snap_utils import read_next_tokenline as token
-from .snap_utils import tryFloat
+
+def finiteFloat(elt):
+    return not (isinf(elt) or isnan(elt))
+
+
+def _ignorable_(line):
```

Comment:
Dont think this (and what uses it) is actually used, must be from the old config.

Reply:
The parsing is flaky and it is used in some constructed test-cases. It ensures that all lines with only comments are skipped. Line-comments are not implemented.